### DISCUSSION TITLE
Where can I get the .config file?

### DISCUSSION THREAD

**Discussion by The-Last-Soldier** [zen-kernel/zen-kernel#321] [Q&A] [answered]
**_Where can I get the .config file?_**

**Comment by damentz** [comment]
What have you tried so far to find the config file?

**Reply by The-Last-Soldier** [reply]
Looking through the git hub files and my personal build of zen

**Reply by damentz** [reply, marked as answer]
Ah I see, so your expectation is that the config file be part of the main sources repository.  That's normally not the case, that's the responsibility of the package maintainer that ships a package for a distribution.  So you'd want to look at the kernel package repo instead, as it will contain the config and any necessary tweaks to get the final desired behavior.

For example, the official and derivative of Zen Kernel:
 1. `linux-zen`: https://gitlab.archlinux.org/archlinux/packaging/packages/linux-zen/-/blob/main/config
 2. `linux-lqx`: https://github.com/damentz/liquorix-package/blob/6.4/master/linux-liquorix/debian/config/kernelarch-x86/config-arch-64
